packages/mask/src/utils/theme/theme-global.d.ts

```diff
@@ -16,6 +26,15 @@ declare module '@mui/material/styles/createPalette.d' {
         input: string
         tipMask: string
         messageShadow: string
+        modalTitle: string
+    }
+}
+
+declare module '@mui/material/index.d' {
+    export interface Color {
+        main: string
+        main2: string
+        main3: string
```

Comment:
This naming is confusing

Reply:
Yep, Just sync with design. I will replace these.